doc/nrf/doc_styleguide.rst

```diff
@@ -57,16 +57,32 @@ You can use different linking and inclusion methods, depending on the content yo
 Hyperlinks
 ----------
 
+**External**:
+
 All external links must be defined in the :file:`links.txt` file.
 Do not define them directly in the RST file.
 This creates one location for validating links, making it easier to update broken links.
 
-Each link should be defined only once in :file:`links.txt`.
+Define each link only once in :file:`links.txt`.
 If you are adding a new link, make sure it has not already been added.
-If the link text that is defined in :file:`links.txt` does not fit in the context where the link will be used, you can override it by using the following syntax: ```new link text <original link text_>`_``.
+If the link text that is defined in :file:`links.txt` does not fit in the context where the link will be used, you can override with the following syntax: ```new link text <original link text_>`_``.
```

Comment:
```suggestion

If the link text that is defined in :file:`links.txt` does not fit in the context where the link will be used, you can override the text with the following syntax: ```new link text <original link text_>`_``.
```
I though adding an empty line might help since we're not talking about using this syntax in the `links.txt` file. Could probably use further clarification.

Also, "override" needs an object.